### ISSUE TITLE
switch, if, if else etc - become null with package

### ISSUE THREAD

**Issue by lewislepton** [snowkit/atom-haxe#35]
the have package is great, but, it kills off things such as say, switch statments, for loops, if statements etc.

finding say openfl stuff is fine, but the actual language for other things dissapear

**Comment by ruby0x1** [comment]
Can you please elaborate what you're comparing to?

The compiler doesn't complete any of these features so I assume you may be referring to features of a different IDE or perhaps some form of snippets?

_(There is no “killing“ of anything happening)_

**Comment by lewislepton** [comment]
it is using atom itself.
but on disabling the atom haxe package, switch statments and if/else etc can then pop up in auto complete. once i enable it, they go away

**Comment by jeremyfa** [comment]
Interesting. It may be the haxe completion provider that mutes the default autocomplete suggestions (including the ones with snippets).

**Comment by ruby0x1** [comment]
They originate from language-haxe @jeremyfa (since they are snippets, and are specific to haxe). 

We can look into why they are consumed though.

**Comment by lewislepton** [comment]
the package i have on are
- atom haxe
- linter
- language haxe
- lime

**Comment by lewislepton** [comment]
ah ok. strange it cuts out though

**Comment by jeremyfa** [comment]
@underscorediscovery Yes, I remember having increased the priority of our plugin completion provider to fix some issues. That said, even with the previous (lower) priority the haxe-language snippets never shown up iirc.
